tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -217,7 +217,6 @@ void hash_algorithm( int alg_arg, int length_arg )
     psa_algorithm_t alg = alg_arg;
```

Comment:
Please add `ALG_IS_SIGN_HASH` to `test_suite_psa_crypto_metadata.function`.

In `test_suite_psa_crypto_metadata.data`, this will mean adding `| ALG_IS_SIGN_HASH` to every place that currently has `ALG_IS_HASH_AND_SIGN` except for `PSA_ALG_RSA_PKCS1V15_SIGN_RAW`.